charts/backingservices/charts/constellation-messaging/README.md

```diff
@@ -0,0 +1,42 @@
+# Constellation messaging service helm chart
+
+The Pega Helm chart is used to deploy an instance of constellation messaging service into a Kubernetes environment.  This readme provides a detailed description of possible configurations and their default values as applicable. 
+
+### Constellation messaging service runtime configuration
+
+The values.yaml file provides configuration options to define the values for the deployment resources of the constellationui service.
+
+### Configuration settings
+
+| Configuration                           | Usage                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+|-----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled`                               | Enables the constellationui service. Set to true to enable constellationui service in the kubernetes environment.                                                                                                                                                                                                               |
```

Comment:
Enables the constellation UI service. Set to true to enable the constellation UI service in Kubernetes environment.